### PR TITLE
Bugfix/avoid memory leaks

### PR DESCRIPTION
**Pull request by t-b**

### CODE REVIEW

**Comment by timjarsky** [comment]
@t-b any guesses as to why the ITC1600 is passing tests now?

File: Packages/MIES/MIES_NeuroDataWithoutBorders.ipf [hunk 1]
```diff
@@ -941,6 +941,10 @@ static Function NWB_ClearWriteChannelParams(s)
 	samplingRate = s.samplingRate
 	groupIndex   = s.groupIndex
 
+	// Clear wave elements before overwriting with a default initialized structure.
+	// This avoids a memory leak. (Reported to WM)
```
Comment (timjarsky) [inline Packages/MIES/MIES_NeuroDataWithoutBorders.ipf]: @t-b I'd like to your thought on this when we chat next.

**Comment by t-b** [comment]
> @t-b any guesses as to why the ITC1600 is passing tests now?

Not really. I kill a few open igor instances though.

File: Packages/MIES/MIES_NeuroDataWithoutBorders.ipf (hunk 1 above)
Comment (t-b) [reply Packages/MIES/MIES_NeuroDataWithoutBorders.ipf]: Sure!

**Pull request merged**